subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -84,6 +84,15 @@ Some builds may have worked around this by also putting the additional attribute
 
 The embedded Kotlin has been updated to link:https://github.com/JetBrains/kotlin/releases/tag/v1.8.21[Kotlin 1.8.21].
 
+[[unsupported_ge_plugin_3.13]]
+==== Earliest supported Gradle Enterprise plugin version is 3.13.1
+
+Starting in Gradle 9.0 the earliest supported Gradle Enterprise plugin version is 3.13.1.
+All earlier versions of the plugin that were previously supported (3.0+) will be ignored when applied.
```

Comment:
```suggestion
All previously supported versions of the plugin (3.0+) will be ignored when applied.
```

Reply:
I think suggestion changes the meaning. I was aiming to describe that among all versions that are currently supported (3.0 up to 3.13.3) *some* of them will become unsupported/ignored (3.0 up to 3.13). Maybe I should just be more explicit.

Reply:
@alllex does my new suggestion work?